docs/docs-content/clusters/edge/site-deployment/site-installation/edge-host-registration.md

```diff
@@ -8,13 +8,12 @@ tags: ["edge"]
 ---
 
 To use an Edge host with a host cluster, you must first register it with Palette. A registration token in the user data
-is required to complete the registration process. You have three options to register the Edge host with Palette.
+is required to complete the registration process. You have two options to register the Edge host with Palette.
```

Comment:
I would advise against counting the number of options here, since this introduces a variable that could be difficult to update in the future. Instead, you could go with something like:
> You have the following options to register the Edge host with Palette.